back/boxtribute_server/graph_ql/definitions/public_api.graphql

```diff
@@ -84,6 +85,35 @@ type TopProductsDimensions {
   category: [DimensionInfo]
 }
 
+type MovedBoxesData implements DataCube {
+  facts: [MovedBoxesResult]
+  dimensions: MovedBoxDataDimensions
+}
+
+"""
+A box can be moved in various ways:
+- within a base (location ID with InStock/Donated)
+- because it's lost (Lost)
+- because it becomes scrap (Scrap)
+- because it's about to be shipped (target base ID with MarkedForShipment)
+- because it's being shipped (target base ID with InTransit/Receiving)
+"""
+type MovedBoxesResult {
```

Comment:
maybe we should add something like a moveType to distinguish between shipments and Box moves